client/src/pages/login.tsx

```diff
@@ -1,13 +1,14 @@
-import { useLogin } from "@refinedev/core";
+import { useLogin } from "@CarRentalServices/core";
 import { useEffect, useRef } from "react";
 
 import Box from "@mui/material/Box";
 import Container from "@mui/material/Container";
 import Typography from "@mui/material/Typography";
-import { ThemedTitleV2 } from "@refinedev/mui";
+import { ThemedTitleV2 } from "@CarRentalServices/mui";
 
 import { CredentialResponse } from "../interfaces/google";
 
+body{background-color:#F2F2F2};
```

Comment:
Try to explore more on the @mui documentation about the components we use in the project and the way we use to style them. Here is a [link](https://mui.com/material-ui/react-box/)!